wiki/content/slash-graphql/slash-quick-start.md

```diff
@@ -134,8 +243,22 @@ Let's try querying back the tasks. We should be getting empty results here, sinc
 }
 ```
 
-## Testing it out with a Simple UI
+We should be getting an empty result this time, since we no longer have access to the tasks.
+
+## Testing it out with a simple UI
+
+To test our work, we've built the app's frontend with [React](https://reactjs.org/), so you can use it to close the tasks off.
+
+Let's head over to our sample React app, deployed at [https://relaxed-brahmagupta-f8020f.netlify.app/](https://relaxed-brahmagupta-f8020f.netlify.app/). When running the React app, remember to use the GraphQL Endpoint shown in your Slash GraphQL [Dashboard](https://slash.dgraph.io/_/dashboard).
+
+You can try creating an account with your email, or log in with the following user/password credentials:
+
+* `frodo@dgraph.io` / `password`
+* `skywalker@dgraph.io` / `password` \(like the first Death Star, Luke wasn't big on security\)
+
+Once you have logged in, you should see something like:
+
+![React](/images/slash-graphql/todos.png)
 
-We've built a todo app with react that you can use to close these todos off. Let's head over to our sample react app, deployed at [https://relaxed-brahmagupta-f8020f.netlify.app/](https://relaxed-brahmagupta-f8020f.netlify.app/).
+Congratulations! You have completed the Slash GraphQL quick-start guide, and you are ready to use the world’s most advanced, hosted GraphQL backend in your applications.
 
-You can try creating an account with your email, or logging in with frodo / skywalker. Like the first death star, Luke wasn't big on security, his password is `password`. Frodo has the same password.
```

Comment:
I'd suggest adding the following section at the end of this document, to provide suggested further reading:

## Next Steps

To learn more about the Slash GraphQL managed service, see [Administering your Backend](https://graphql.dgraph.io/slash-graphql/admin/) and [Advanced Queries](https://graphql.dgraph.io/slash-graphql/advanced-queries).

Reply:
fixed